### PR TITLE
Add latest enabled build to source files

### PR DESCRIPTION
**Pull request by orangejenny**
Request from Marissa.

screen shot 2019-03-04 at 3 35 09 pm

@millerdev / @mkangia 

Product: this is for the RELEASE_BUILDS_PER_PROFILE flag ("Do not release builds for all app profiles by default. Then manage via Source files view").

### CODE REVIEW

**Review by mkangia — CHANGES_REQUESTED** [review]
The latest enabled build profiles are already visible on the app profiles listing. I think fine to keep it at both places if that looks better, plus we can't remove the method `get_enabled_build_profiles_for_version` because it's used there?

**Comment by orangejenny** [comment]
Yeah, they're on the releases page, this came up because of a complaint about flipping back and forth between the two tabs being a headache.

> we can't remove the method get_enabled_build_profiles_for_version because it's used there

I don't follow - where is this still used?

**Comment by mkangia** [comment]
> between the two tabs being a headache

too much comfort.

You are right. I thought it was used in that listing of profiles but i was wrong.

**Review by mkangia — APPROVED** [review]

File: corehq/apps/app_manager/templates/app_manager/download_index.html [hunk 1]
```diff
@@ -13,28 +13,42 @@
...
+                    </td>
+                    <td class="col-sm-2">
+                        {% if build_profile_id in enabled_build_profiles %}
+                            <a class="btn btn-default" href='{% url "toggle_build_profile" app.domain app.id build_profile_id %}?action=disable'>
+                                <i class="fa fa-chain-broken"></i>
+                                {% trans 'Revoke restriction' %}
+                            </a>
+                        {% else %}
+                            <a class="btn btn-default" href='{% url "toggle_build_profile" app.domain app.id build_profile_id %}?action=enable'>
+                                <i class="fa fa-ban"></i>
```
Comment (mkangia) [inline corehq/apps/app_manager/templates/app_manager/download_index.html]: nice buttons decorations. 👍

Comment (orangejenny) [reply corehq/apps/app_manager/templates/app_manager/download_index.html]: 🙇‍♀️ so much of my time is spent looking at the fontawesome cheat sheet...

**Pull request merged**